web/components/templates/sessions/sessionsPage.tsx

```diff
@@ -90,7 +101,7 @@ const SessionsPage = (props: SessionsPageProps) => {
 
   // State for active columns
   const [activeColumns, setActiveColumns] = useState<DragColumnItem[]>(
-    columnDefsToDragColumnItems(INITIAL_COLUMNS)
+    columnDefsToDragColumnItems(getColumns())
```

Comment:
```suggestion
     const [selectedData, setSelectedData] = useState<TSessions | undefined>(
    undefined
  );
  // State for active columns
  const [activeColumns, setActiveColumns] = useState<DragColumnItem[]>(
    columnDefsToDragColumnItems(
      (() => {
        return Object.keys(selectedData?.metadata ?? {}).map((property) => {
          return {
            id: property,
            accessorFn: (row: any) => {
              const value = row.metadata ? row.metadata[property] : "";
              return value;
            },
            cell: ({ row }: any) => {
              return row.original.metadata[property];
            },
          };
        });
      })()
    )
  );
```

Get get rid of the other function?

Reply:
and also delete that file lol